easybuild/easyconfigs/i/intel/intel-2017b.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'Toolchain'
+
+name = 'intel'
+version = '2017b'
+
+homepage = 'http://software.intel.com/en-us/intel-cluster-toolkit-compiler/'
+description = """Intel Cluster Toolkit Compiler Edition provides Intel C/C++ and Fortran compilers, Intel MPI &
+ Intel MKL."""
```

Comment:
Wouldn't it be better to have a proper toolchain description here?

Reply:
Something like this then?

```
Compiler toolchain including Intel compilers, Intel MPI and Intel Math Kernel Library (MKL).
```

Reply:
Yes, that's much better.